src/test/java/de/tum/in/www1/artemis/lecture/TextUnitIntegrationTest.java

```diff
@@ -69,6 +69,7 @@ void testAll_asStudent() throws Exception {
     void createTextUnit_asEditor_shouldCreateTextUnitUnit() throws Exception {
         var persistedTextUnit = request.postWithResponseBody("/api/lectures/" + this.lecture.getId() + "/text-units", textUnit, TextUnit.class, HttpStatus.CREATED);
         assertThat(persistedTextUnit.getId()).isNotNull();
+        assertThat(persistedTextUnit.getName()).isEqualTo(textUnit.getName().strip());
```

Comment:
We should assert actual values in the test cases instead of duplicating production logic.
```suggestion
        assertThat(persistedTextUnit.getName()).isEqualTo("LoremIpsum");
```